incubator/kafka/values.yaml

```diff
@@ -7,9 +7,9 @@ Image: "solsson/kafka"
 ImageTag: "0.11.0.0"
 ImagePullPolicy: "IfNotPresent"
 Cpu: "100m"
-Memory: "512Mi"
+Memory: "1024Mi"
```

Comment:
I think we are moving away from default resources instead they are entered like below, 
```
 resources: {}
    # limits:
    #   cpu: 10m
    #   memory: 20Mi
    # requests:
    #   cpu: 10m
    #   memory: 20Mi
```
anyways why are we changing the default values now?